packages/helpermodules/update_config.py

```diff
@@ -1532,3 +1534,13 @@ def upgrade_datastore_44(self) -> None:
         except Exception:
             log.exception(f"Logdatei '{filepath}' konnte nicht konvertiert werden.")
         self.__update_topic("openWB/system/datastore_version", 45)
+
+    def upgrade_datastore_45(self) -> None:
+        def upgrade(topic: str, payload) -> Optional[dict]:
+            if re.search("^openWB/general/chargemode_config/phase_switch_delay$", topic) is not None:
```

Comment:
```suggestion
            if re.search("^openWB/general/chargemode_config/pv_charging/phase_switch_delay$", topic) is not None:
```
Die if-Abfrage sucht nach dem alten Topic.

Reply:
wurde angepasst. Danke